### PR TITLE
[t1970] Added tooltips

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: src/layouts/ui-kit.html [hunk 1]
```diff
@@ -8,8 +8,9 @@
       <div class="editor-options scrollbar-inner">
         <fieldset>
           <label>Buttons</label>
-          <a class="butter-btn btn-light">Button</a>
-          <a class="butter-btn btn-green">Button</a>
+          <a class="butter-btn btn-light">Button
+          </a>
```
Comment (jbuck) [inline src/layouts/ui-kit.html:12]: this `<a/> is pretty lonely all by itself. Lets put it back

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/6e19638e7e15d48/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/6e19638e7e15d48/output.txt

Total script time: 1.18 mins

**Pull request closed without merging**